### ISSUE TITLE
Full dataset missing from Download tab for unreleased studies in QA

### ISSUE THREAD

**Issue by nkittur-uga** [VEuPathDB/web-eda#1047]
On qa.restricted.clinepidb.org, in all the unreleased studies (e.g., GAMIN, MORDOR, Group Antenatal Care...), the download tab shows "My subset" but the "Full Dataset" portion is completely missing. 
<img width="1301" alt="Screen Shot 2022-03-09 at 5 24 42 PM" src="https://user-images.githubusercontent.com/67710160/157551524-1bcf7706-fcf2-49ee-9d89-b0281250174e.png">

**Comment by SheenaTomko** [comment]
I added an entry including a date to b22 in eupathBuildDates.yml. We can see if having the entry there ensures an entry in the  DataVersions table and fixes the issue. Check back tomorrow following the rebuild.

**Comment by SheenaTomko** [comment]
@jaycolin can you take a look at this? See Dave's comment above about the DataVersions table being empty for GAMIN. We're also having the same issue with Preventing Congenital Syphilis, Myanmar ICEMR, MORDOR, G-ANC, etc.)
My attempted fix, adding a date for b22 in the eupathBuildDates.yml didn't seem to work.

**Comment by dmfalke** [comment]
I'm tagging @aurreco-uga on this issue. She might have some insight into what's happening.

As a recap, the issue is that the `DataVersions` table for these studies is empty. They all(?) have `<history buildNumber="22">` in the presenters file. I noticed that UMSP also has a history entry for build 22, but it does not appear in the download tab, so I suspect this build number is being excluded for some reason. The model currently says the current build is "22", fwiw.

**Comment by aurreco-uga** [comment]
the only problem is that the versions tables requires a tuning table that does not get generated by the dsp jenkins job....  ill try to figure out how to run it.. that should fix it..

**Comment by nkittur-uga** [comment]
Yes, full datasets are now available for unreleased studies. QA is complete!
